src/components/SavedProvidersList/saved-providers-list.js

```diff
@@ -50,6 +50,27 @@ function printSavedProviders(providers) {
   printJSX(printPage);
 }
 
+function emailSavedProviders(providers) {
+  const byTypeName = _.groupBy(providers, provider => provider.typeName);
+  let emailBodyString = ''
+  const newLine = "%0D%0A"
+  _.forEach(byTypeName, (providers, typeName) => {
+    let providerString = ''
+    providerString = providerString.concat(typeName + newLine)
+    providers.forEach((provider) => {
+      providerString = providerString.concat(
+        newLine + provider.name + newLine +
+        "Address: " + provider.address + newLine +
+        "Website: " + provider.website + newLine +
+        "Phone: " + provider.telephone + newLine +
+        "Email: " + provider.email + newLine)
+      })
+    emailBodyString = emailBodyString.concat(providerString + newLine)
+  })
+  // Opens up default email application populated with formatted provider data
+  window.location.href = "mailto:?&body="+emailBodyString;
```

Comment:
What do you think about replacing this with window.open(...)? That will open the email in a separate tab rather than replacing the current page, which is extra important here because we don't actually save saved list between page visits.